src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+type Color = 'red'| 'green'| 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
better create enum for Shape, then you can use it like Shape.Triangle or other in classes